docs/runbook/upgrade-repository-dependencies/README.md

```diff
@@ -0,0 +1,131 @@
+# Upgrade the project's dependencies in the repository
+
+## Introduction
+
+This runbook provides step-by-step instructions to upgrade the dependencies in the repository, including Rust crates, documentation, and JavaScript packages.
+
+## Steps
+
+### Update Rust dependencies
```

Comment:
Can you add a **Upgrade Rust outdated dependencies** section, explaining how to use [Dependi](https://www.dependi.io/) to identify and attempt to updated outdated dependencies (which can introduce breaking changes in our code)?

Reply:
Done ✅, let me know if that works for you: https://github.com/input-output-hk/mithril/pull/1893/commits/458391bb5245675954c96c1295ec6926cb9311b5